lib/types/youtube.js

```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```

Comment:
Hm....

Now I learnt something new. Youtube can be different aspect ratio.

https://www.youtube.com/watch?v=dt-dDg6uod0 can be shared at `640 X 360` while https://www.youtube.com/watch?v=sUg401QLa7I can be shared at `640 X 480`.

I'd opt for `640 x 480` to be the default - but I'm interested in hearing your thoughts.